src/java.base/share/classes/java/lang/ref/Cleaner.java

```diff
@@ -87,8 +87,12 @@
  * <pre>{@code
  * public class CleaningExample implements AutoCloseable {
  *        // A cleaner, preferably one shared within a library
- *        private static final Cleaner cleaner = <cleaner>;
+ *        private static final Cleaner cleaner = Cleaner.create();
  *
+ *        // Static state class, capturing information necessary for
+ *        // State class captures information necessary for cleanup.
```

Comment:
I think it would be simpler to just drop L92, meaning start with "State class captures ..." rather than having "state class" and "State class" in the same sentence.

Reply:
I have removed that line and also changed `in this CleaningExample` -> `in this example`, because it is obvious.